invenio_users_resources/services/permissions.py

```diff
@@ -15,21 +15,30 @@
     SystemProcess,
 )
 
-from .generators import IfPublicEmail, IfPublicUser, Self
+from .generators import IfPublicEmail, IfPublicUser, Self, UserModeration
 
 
 class UsersPermissionPolicy(BasePermissionPolicy):
     """Permission policy for users and user groups."""
 
     can_create = [SystemProcess()]
-    can_read = [IfPublicUser([AnyUser()], [Self()]), SystemProcess()]
+    can_read = [
+        UserModeration(),
+        IfPublicUser([AnyUser()], [Self()]),
```

Comment:
if the profile of the user is public then they should see everything no?

Reply:
They should not see everything I think. These "system" fields are something different.

See: 

https://github.com/inveniosoftware/invenio-users-resources/pull/91#discussion_r1280410272